sidebars.js

```diff
@@ -22,10 +22,16 @@ const sidebars = {
     },
     {
       type: "category",
-      label: "Recipes",
+      label: "Infinite Red Recipes",
       collapsed: false,
       items: [{ type: "autogenerated", dirName: "recipes" }],
     },
+    {
+      type: "category",
+      label: "Community Recipes",
```

Comment:
I think a drawback of structuring it this way is people are very unlikely to see it, since the IR recipes category starts expanded.

That said I'm good with starting here and we can iterate on it later.

Off the top of my head, maybe one of the easiest options would be to have "Infinite Red Recipes" and "Community recipes" each link to their own index page (looks like https://docusaurus.io/docs/category/guides), without the sub-categories there, but I couldn't get that working quickly. Docusaurus doesn't seem to be able to even make that auto-generated index if we're not listing it under the sidebar, so I think we'd have to build something custom.

Reply:
I went with this at first because it extends the pattern we made with `Archives` but I'm happy to look into alternative solutions too!

Reply:
But as a first pass, especially with the new search (woot woot), I think this should be fine for now.